tests/keithley_2700.py

```diff
@@ -16,17 +21,23 @@
     },
 ]
 
-TEST_MODES = [TestModes.RECSIM, TestModes.DEVSIM]
+# TEST_MODES = [TestModes.RECSIM, TestModes.DEVSIM]
+TEST_MODES = [TestModes.DEVSIM]
 
 on_off_status = {False: "OFF", True: "ON"}
 
+DRIFT_TOLERANCE = 0.1
```

Comment:
Might be appropriate to have a tolerance on the same order of magnitude as the drift values.

Reply:
Needs refining once the new interpolation routine is in place